os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -789,12 +789,7 @@ int8_t cmd_wifi_off(void)
 		}
 	}
 	RTW_API_INFO("\n\rWIFI Mode Change instead of WIFI reload\r\n");
-
-#if 0
-	if (!wifi_off())
-		return 0;
-	return -1;
-#endif
+	return 0;

```

Comment:
And this is not related to this PR, but it is better to add `#ifndef CONFIG_ENABLE_P2P` for the variable `mode` in line 766, because that is only used when `CONFIG_ENABLE_P2P` is not enabled.

Reply:
I think it's better. however it's fixed by realtek wi-fi driver engineer.

Reply:
Yes this way is better, or can move "int mode;" after line 778. Do you suggest me submitting a new pr for this modification? Thank you.